applications/GeoMechanicsApplication/custom_elements/small_strain_U_Pw_diff_order_element.cpp

```diff
@@ -357,10 +357,11 @@ void SmallStrainUPwDiffOrderElement::SetValuesOnIntegrationPoints(const Variable
     KRATOS_TRY
 
     if (rVariable == CAUCHY_STRESS_VECTOR) {
-        KRATOS_ERROR_IF(rValues.size() != mStressVector.size())
+        KRATOS_ERROR_IF(rValues.size() != this->GetGeometry().IntegrationPointsNumber(mThisIntegrationMethod))
```

Comment:
Nitpicking: I think we don't need the explicit `this->` here:
```suggestion
        KRATOS_ERROR_IF(rValues.size() != GetGeometry().IntegrationPointsNumber(mThisIntegrationMethod))
```

Reply:
Good point, this element isn't templated (yet?), removed!